.github/workflows/gh-pages.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - run: pip install mkdocs==1.2.4 mkdocs_material==8.1.9
       - run: mkdocs build
       - run: make parse-examples
-      - uses: peaceiris/actions-gh-pages@v2.9.0
+      - uses: peaceiris/actions-gh-pages@v3.8.0
```

Comment:
??

Reply:
I didn't touch this yaml 🤔  It's a bit weird but I'm reverting it.